### PR TITLE
[collector] #686 - Ensure ClusterRole is created for kubeletMetrics preset

### PR DESCRIPTION
**Pull request by cjbhaines**
Addresses: https://github.com/open-telemetry/opentelemetry-helm-charts/issues/686

### CODE REVIEW

**Comment by Allex1** [comment]
@cjbhaines you need to sign the CLA, bump the chart version and then run `make generate-examples` in order for the CI to pass. [GUIDE](https://github.com/open-telemetry/opentelemetry-helm-charts/blob/main/charts/opentelemetry-operator/CONTRIBUTING.md)

**Comment by cjbhaines** [comment]
> @cjbhaines you need to sign the CLA, bump the chart version and then run `make generate-examples` in order for the CI to pass. [GUIDE](https://github.com/open-telemetry/opentelemetry-helm-charts/blob/main/charts/opentelemetry-operator/CONTRIBUTING.md)

Thanks @Allex1, that's done.

File: charts/opentelemetry-collector/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: opentelemetry-collector
-version: 0.50.0
+version: 0.51.0
```
Comment (TylerHelmuth) [inline charts/opentelemetry-collector/Chart.yaml:3]:
```suggestion
version: 0.50.1
```
We can get away with only a patch bump for this bug fix.  Will need to re-run `make generate-examples CHARTS=opentelemetry-collector`.

**Review by TylerHelmuth — COMMENTED** [review]
Thanks for fixing this bug!

File: charts/opentelemetry-collector/Chart.yaml (hunk 1 above)
Comment (cjbhaines) [reply charts/opentelemetry-collector/Chart.yaml]: My pleasure! That's done, thanks.

**Review by TylerHelmuth — APPROVED** [review]

**Pull request merged**